### PR TITLE
importinto: support split large csv file

### PR DESCRIPTION
**Pull request by D3Hunter**
### What problem does this PR solve?

Issue Number: ref #42930

Problem Summary:

### What is changed and how it works?
- support `split_file` option for csv files, and add restriction that skip_rows should be 0 or 1 when `split_file` is enabled.

### Check List

Tests 

- [x] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Comment by D3Hunter** [comment]
@lance6716 @lichunzhu

**Review by okJiang — APPROVED** [review]

**Review by lance6716 — APPROVED** [review]

**Comment by lance6716** [comment]
/hold

need address comment

File: executor/importer/import.go [hunk 1]
```diff
@@ -459,7 +462,6 @@ func (e *LoadDataController) checkFieldParams() error {
 		}
 		// NOTE: IMPORT INTO also don't support user set empty LinesTerminatedBy or FieldsTerminatedBy,
 		// but it's check in initOptions.
-		// TODO: support lines terminated is "".
```
[the remark this answers is not in the document]
Comment (D3Hunter) [reply executor/importer/import.go:462]: i think this is more like a feature, not a `todo` item.

**Comment by D3Hunter** [comment]
@lance6716

**Pull request merged**